homeassistant/components/jewish_calendar/sensor.py

```diff
@@ -1,31 +1,133 @@
 """Platform to retrieve Jewish calendar information for Home Assistant."""
+from __future__ import annotations
+
 from datetime import datetime
 import logging
 
 import hdate
 
-from homeassistant.components.sensor import SensorEntity
+from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
 from homeassistant.const import DEVICE_CLASS_TIMESTAMP, SUN_EVENT_SUNSET
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.sun import get_astral_event_date
+from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
 import homeassistant.util.dt as dt_util
 
-from . import DOMAIN, SENSOR_TYPES
+from . import DOMAIN
 
 _LOGGER = logging.getLogger(__name__)
 
-
-async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+DATA_SENSORS = (
+    SensorEntityDescription(key="date", name="Date", icon="mdi:judaism"),
```

Comment:
If you add the last comma (to all of them) and re-run black, they all will be formatted the same making it a bit more readable.
```suggestion
    SensorEntityDescription(key="date", name="Date", icon="mdi:judaism",),
```

Reply:
thanks!